### ISSUE TITLE
Downloading of updated audio files

### ISSUE THREAD

**Issue by ismail-s** [quran/quran_android#1374]
This issue is spun off from #204 (specifically, from https://github.com/quran/quran_android/issues/204#issuecomment-637129112). From the issue I encountered and described in that comment, I think that the audio file for Surah 28 for Sudais got updated, and only the timing database got updated, not the mp3 file.

@ahmedre does the app check for updated audio files currently (I have tried to look through the code a bit to check this, but there's a lot of code to look through and I find it hard to follow)? If yes, how? If not, then I think we should add some sort of check for this.

**Comment by ahmedre** [comment]
the audio file for sura 28 was also updated -see [here](https://download.quranicaudio.com/quran/abdurrahmaan_as-sudays/)

<img width="807" alt="Screen Shot 2020-06-07 at 12 24 29 AM" src="https://user-images.githubusercontent.com/47771/83953874-3dc18280-a855-11ea-846e-12205280ce17.png">


i just tested and it works fine.

yes, the app checks weekly for updates - it does as of the last release (it was implemented in #1318). not sure if there's a bug, but it should remove both the mp3 and database (when necessary since not all updates require database upgrades) for the update to go through.

**Comment by nacer80** [comment]
Salam 3alikom 
I checked it we haven't any issue with the audio and database files, I think our issue in app script. wa Allaho a3lem.

**Comment by ahmedre** [comment]
wa3laikum alsalam wa ra7matullah - 
really sorry about the late reply - but looks like you're right @ismail-s jazakumAllah khairan - querying the [audio updates api](https://android.quran.com/data/audio_updates.php), i see (for sheikh Sudais):

```json
        {
            "database_version": 2,
            "files": [
                {
                    "filename": "002.mp3",
                    "md5": "87255b9c8be31672eae0f408b1af6e35"
                },
                {
                    "filename": "027.mp3",
                    "md5": "56dc552768c19b995618525be6a0de6f"
                },
                {
                    "filename": "058.mp3",
                    "md5": "f1c11d38d1c6ddd4cc5224f7da0096b8"
                }
            ],
            "path": "https://download.quranicaudio.com/quran/abdurrahmaan_as-sudays/"
        },
```

in my notes i also have:

```
### Sudais
* gapless - replaced suras Baqarah, Naml, Qasas, and Mujadala to improve quality of all the aforementioned - gapless only - 3/7/2020

#### Additional Notes
* Database version 2 pushed on -3/7/2020 to update the replaced gapless copies of suras 2, 27, 28, and 58.
```

so yeah looks like i forgot sura Qasas :(

**Comment by ismail-s** [comment]
@ahmedre Jazakallah for following this up.

**Comment by ahmedre** [comment]
fixed now - jazakumAllah khairan.
